examples/tutorial_examples/baby_afq.py

```diff
@@ -0,0 +1,154 @@
+"""
+=============================================
+BabyAFQ : tractometry for infant dMRI data
+=============================================
+
+The following is an example of tractometry for infant bundles. This example and
+resulting pyAFQ support for pediatric bundles was inspired by and largely due
+to the work of Grotheer et al. [Grotheer2022]_, as implemented in
+[Grotheer2023]_.
+
+.. note::
+    Because it is time and disk-space consuming, this example
+    is not run when the pyAFQ documentation is built. To run this example
+    yourself, you can download the contents of this file as an
+    executable `.py` file or as a Jupyter notebook from the links at the bottom
+    of the page.
+
+"""
+import os.path as op
+import plotly
+import wget
```

Comment:
Even development version of pyAFQ does not include wget module as part of package. Will need to specify somewhere or new user will be unable to import wget directly.